BlockSettleUILib/Trading/WalletShieldBase.cpp

```diff
@@ -250,5 +252,9 @@ void WalletShieldBase::showShieldGenerateAuthAddress()
 
 void WalletShieldBase::showShieldAuthValidationProcess()
 {
-   showShield(shieldAuthValidationProcessText, shieldButtonView, shieldAuthValidationProcessHeader);
+   const bool isProd = appSettings_->get<int>(ApplicationSettings::envConfiguration) ==
+      static_cast<int>(ApplicationSettings::EnvConfiguration::Production);
+
+   showShield(shieldAuthValidationProcessText.arg(isProd ? tr("24h") : tr("15minutes")),
```

Comment:
Better to put space between "15" and "minutes"

Reply:
Space added, thank you